src/Metadata/GraphQl/Mutation.php

```diff
@@ -37,7 +33,7 @@ public function __construct(
         ?bool $paginationClientPartial = null,
         ?bool $paginationFetchJoinCollection = null,
         ?bool $paginationUseOutputWalkers = null,
-        ?bool $paginationViaCursor = null,
+        ?array $paginationViaCursor = null,
```

Comment:
we need to check https://github.com/api-platform/core/pull/5333 probably work on this target branch instead

Reply:
Not sure about #5333’s state; I’ll rebase this one when merged.

Reply:
I merged it!